### ISSUE TITLE
What does (pending) mean?

### ISSUE THREAD

**Issue by tonyxiao** [kamilkisiela/graphql-config#6]
I see a number of editors have support = pending.

Does that mean I can try them? If so how?

**Comment by tonyxiao** [comment]
On a related note, how exactly do I actually install this tool? Set an env var won't do anything since graphql-config isn't installed yet.

**Comment by schickling** [comment]
Pending means the integration work for the specific tool is still work-in-progress. Once the work is done for a tool you don't need to install graphql-config explicitly as it comes baked in with the tool.

**Comment by tonyxiao** [comment]
Got it, any way for me to help contribute to get something working for eslint-plugin-graphql? Could you maybe give me some pointers on how to proceed?

**Comment by schickling** [comment]
Sure! Unfortunately I'm pretty busy at this moment but I already began with the integration for eslint: https://github.com/schickling/eslint-plugin-graphql

Maybe you could take it from there?

**Comment by tonyxiao** [comment]
Hmm your branch seems to be already working, what more is there to do/

**Comment by schickling** [comment]
I think @stubailo wanted to avoid a breaking change and therefore allow to include the schema the "old way". The problem was error handling. It currently [throws an error](https://github.com/apollostack/eslint-plugin-graphql/blob/master/src/index.js#L65) if you don't provide the `schemaJson` option which gets evaluated every time the linter runs. graphql-config on the other hand should just do the parsing (and schema fetching) once since it's blocking and could slow down the linter otherwise.

Maybe a strategy could be to `try` to parse the graphql-config and force the "old way" if that failed. Would that make sense to you?

**Comment by tonyxiao** [comment]
I see. How would you implement "do the parsing (and schema fetching) once"? Would you save the result to some intermediate file? Or is linter a persistent process where you can keep it in memory?

**Comment by schickling** [comment]
Unfortunately I don't have too much insight into how eslint works internally. I could imagine caching the schema in a local variable could do the trick?

**Comment by danielrearden** [comment]
Closing since the original question was answered. [eslint-plugin-graphql](https://github.com/apollographql/eslint-plugin-graphql) is available and currently uses the latest version of `graphql-config`.
